src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -308,10 +321,28 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
             string line;
             var sb = new StringBuilder();
 
+            bool mitigationMissingCommandLineUnknownOnly = false;
+
+            if (mitigationUnknownNoCommandLineRaw.Count > 0)
+            {
+                mitigationMissingCommandLineUnknownOnly = true;
+
+                // The following modules were compiled with a toolset that supports
+                // /Qspectre but no compiler command line is present and therefore
+                // it cannot be determined if /Qspectre was specified: {0}
+                line = string.Format(
+                RuleResources.BA2024_Warning_SpectreMitigationUnknownNoCommandLine,
+                        mitigationUnknownNoCommandLineRaw.CreateOutputCoalescedByLibrary());
+                sb.AppendLine(line);
+            }
+
             if (mitigationExplicitlyDisabledModules.Count > 0)
             {
-                // The following modules were compiled with Spectre
-                // mitigations explicitly disabled: {0}
+                mitigationMissingCommandLineUnknownOnly = false;
+
+                // The following modules were compiled with a toolset that supports
```

Comment:
>following [](http://example.com/codeflow?start=23&length=9)

Is this *really* the string we report? Because if so, it needs improvement, let me know I will go edit.

Reply:
These were the existing messages.
This shows up as new because there were no comments above the messages so I put them there. That's true for multiple of these.